distribution/lib/Standard/Table/0.0.0-dev/src/Delimited/Reader.enso

```diff
@@ -35,17 +35,17 @@ polyglot java import org.enso.table.read.QuoteStrippingParser
      operation. By default, a warning is issued, but the operation proceeds.
      If set to `Report_Error`, the operation fails with a dataflow error.
      If set to `Ignore`, the operation proceeds without errors or warnings.
-read_file : Delimited -> File -> Problem_Behavior -> Any
-read_file format file on_problems =
+read_delimited_file : Delimited_Format -> File -> Problem_Behavior -> Any
+read_delimited_file format file on_problems =
```

Comment:
I'm a bit unsure about these renames, it is a stylistic convention so I'm not gonna argue too much, it is ok too.

But what about not importing the functions directly into scope but using them as qualified?
For example I think we could even do:
```
import Standard.Table.Delimited

...
Delimited.Reader.read_file ...
```

I really like the look of that and there is actually _less_ scope pollution.

Although the current one is ok too, I think I just wanted to put another one on the table so we can choose one.